ArkKit/Ark.swift

```diff
@@ -14,9 +14,14 @@ class Ark {
     let rootView: UINavigationController
     let eventManager = ArkEventManager()
     let ecsManager = ArkECS()
+    let gameScene: AbstractArkGameScene // Ideas for naming this please, maybe physics scene but it sounds weird
```

Comment:
General FYI:
- I think we will / should move hiding systems & adding of systems as a callback within `ArkBlueprint`. 
- So there should be a `setUpSystem` of sorts that runs. Think @chownces is working on this. So this would likely move to the blueprint level within the callback.